README.md

```diff
@@ -1 +1,50 @@
-# anycloud
+# AnyCloud
+
+AnyCloud is an Lambda alternative for Node.js that works with any cloud provider. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
```

Comment:
My biggest gripe with Lambda was not being sure what version of any lambda endpoint was actually deployed, and the difficulties of automating removal of dead lambda functions. Not sure how to turn that into a simple bullet point, though.

Reply:
hmmm good point